src/components/tabs/components/tab/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from 'react'
+import { TabProps } from './types'
+
+const Tab = (props: TabProps): ReactElement => {
```

Comment:
nit, but from what I can tell (and I might have the wrong idea), we never really render this component, right? If that's the case, maybe we can add a quick comment here:

```suggestion
/**
 * Note that we never actually render the `Tab` component,
 * we've set up this component for authoring convenience in MDX.
 */
const Tab = (props: TabProps): ReactElement => {
```

... or if I'm off on this, definitely feel free to drop this suggestion & resolve 👍

Reply:
This is a good idea! I think adding a `console.warn` inside the component might be good as well, I'm going to test that real quick before adding. But will definitely add the comment above the component!

Reply:
Updated in: https://github.com/hashicorp/dev-portal/pull/104/commits/7f0bcd5d2d1c67a0b6a34b803b55bcda798f3d8b